### PR TITLE
ironbank: update the list of maintainers for the ironbank

### PR DESCRIPTION
**Pull request by v1v**
## Proposed commit message

Update the list of maintainers for the ironbank, so it reflects the new org.

## Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have made corresponding change to the default configuration files
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.

## Author's Checklist

- [ ]

## How to test this PR locally

```bash
cd x-pack/metricbeat
mage package
make -C ironbank package
mage ironbank
```

Then you will see the `.tar.gz` in the `build/distributions` folder for each subscribed beat, and `hardening_manifest.yaml` will contain the new up-to-date list of maintainers

## Related issues

https://github.com/elastic/beats/pull/32251 was the initial implementation

### CODE REVIEW

**Review by amannocci — APPROVED** [review]

**Comment by v1v** [comment]
@jlind23 @cmacknz , those users are the ones that will receive notifications regarding the Ironbank release process. You can include as many new maintainers as you wish. 

I plan to merge this as soon as I get another approval.

**Review by cmacknz — APPROVED** [review]

**Pull request merged**
